docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[cases-api-find-cases]]
+== Find cases API
+++++
+<titleabbrev>Find cases</titleabbrev>
+++++
+
+Retrieves a paginated subset of cases. By default, the first page is returned
+with 20 results per page.
+
+=== Request
+
+`GET <kibana host>:<port>/api/cases/_find`
```

Comment:
We should also document the path that takes a space ID, and document that if the space is not provided in the URL, the default space is used.

```
GET <kibana host>:<port>/s/<space_id>/api/cases/_find
```

Reply:
Thanks!  I've made that change in 1f36fd9da172bf319fa6acd1b31a11cf46624d9a